doc/release/contribs.py

```diff
@@ -69,11 +69,10 @@ def get_user(login):
     return user
 
 
-def get_merged_pulls(user, repo, date, page=1, results=0):
+def get_merged_pulls(user, repo, date, page=1, results=0, branch='master'):
     # See https://developer.github.com/v3/search/#search-issues
     # See https://help.github.com/articles/understanding-the-search-syntax/#query-for-dates
-    query = 'repo:{user}/{repo} merged:>{date} sort:created-asc'
-    query = query.format(user=user, repo=repo, date=date)
+    query = f'repo:{user}/{repo} merged:>{date} sort:created-asc base:{branch}'
```

Comment:
This kind of stuff doesn't work for python 3.5. it means you need 3.6 to run. not really LTS friendly.....

Reply:
Only developers run these scripts, so I don't think it matters.